File: src/URI/URI_1066.java
package URI;
import java.util.Scanner;

public class URI_1066 {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		
		int totPositivo = 0;
		int totNegativo = 0;
		int totImpar = 0;
		int totPar = 0;
		
		for(int i = 0; i < 5; i++) {
			int n = sc.nextInt();
			if(n % 2 != 0) {
				totImpar += 1;				
			}
			else {
				totPar += 1;				
			}
			if(n < 0) {
				totNegativo += 1;
			}
			else if(n > 0) {
				totPositivo += 1;
			}
					
		}					
			
		System.out.println(totPar + " valor(es) par(es)");
		System.out.println(totImpar + " valor(es) impar(es)");
		System.out.println(totPositivo + " valor(es) positivo(s)");
		System.out.println(totNegativo + " valor(es) negativo(s)");
		
		
		sc.close();

	}

}

File: kotlin/URI_1066.kt
import java.util.*

fun main(args: Array<String>) {
    val sc = Scanner(System.`in`)
    var pares = 0
    var impares = 0
    var positivos = 0
    var negativos = 0
    for (i in 1..5) {
        val n = sc.nextInt()
        if (n % 2 == 0) pares++
        else impares++
        if (n > 0) positivos++
        else if (n < 0) negativos++
    }
    println("$pares valor(es) par(es)")
    println("$impares valor(es) impar(es)")
    println("$positivos valor(es) positivo(s)")
    println("$negativos valor(es) negativo(s)")
}
